app/controllers/podcast_episode_appearances_controller.rb

```diff
@@ -0,0 +1,114 @@
+class PodcastEpisodeAppearancesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create edit update destroy]
+  before_action :set_episode, only: %i[new]
+  before_action :set_podcast_episode_appearance, only: %i[show edit update destroy]
+  before_action :set_episode_with_strong_params, only: %i[create update]
+  before_action :set_podcast_episode_appearances, only: %i[index create update destroy]
+
+  def index; end
```

Comment:
In this case for example I imagine these will be listed in the podcast page, for example https://dev.to/salesforce_developers_podcast/060-integration-solutions-with-nina-rosen

So when I'm checking the podcast episode I can see the names of the Forem users that have appeared on it.

Thus, I don't think we need a separate index controller. We just need to load the list of users that appeared in the episode to be listed in the episode  page

Reply:
Hi @rhymes, I was busy these days. This is not what the final version is (this index action is created at this point just for testing). What do you think about these notes:

1. probably appearance views can be created as the new folder under podcast episodes views - podcast episode appearances are strictly related to episodes (this is part of another issue actually)
2.index and create action should be part of podcast episode controller
3. update and delete action - keep them in the podcast episode appearances controller.
4. for create action - Ben suggested to provide minimal functionality at this point - enable adding guests of podcast episodes (input = list of comma-separated usernames,  role = `guest`)